qiskit_experiments/calibration_management/parameter_value.py

```diff
@@ -69,6 +69,8 @@ def __post_init__(self):
                     f"Cannot parse {self.date_time} in either of {formats} formats."
                 )
 
+        self.date_time = self.date_time.astimezone()
```

Comment:
I have not done much with time zones. In `save()` we use `%z` which stores a utc offset. I am just wondering how that compares to a time zone. If you save and reload calibrations, do you get close enough to the same datetime object -- date time with utc offset vs. date time with time zone?

Reply:
I think we are okay. I'm not so familiar with time zones either. The following
```python
from datetime import datetime
new_date = datetime.strptime("16/09/20 10:21:35.012+0200", "%d/%m/%y %H:%M:%S.%f%z")
print(new_date)
print(new_date.astimezone())
print(new_date.astimezone() == new_date)
```
prints
```
2020-09-16 10:21:35.012000+02:00
2020-09-16 10:21:35.012000+02:00
True
```

Reply:
Cool. Here is my output:

```
2020-09-16 10:21:35.012000+02:00
2020-09-16 04:21:35.012000-04:00
True
```

So the utc offset changes but equality holds. It is also interesting to look at the repr without and with astimezone:

```python
datetime.datetime(2020, 9, 16, 10, 21, 35, 12000, tzinfo=datetime.timezone(datetime.timedelta(seconds=7200)))
```
vs
```python
datetime.datetime(2020, 9, 16, 4, 21, 35, 12000, tzinfo=datetime.timezone(datetime.timedelta(days=-1, seconds=72000), 'EDT'))
```
The timezone version has a timezone label attached but as long as we don't look at that and just deal with datetime objects I think we are okay.